examples/gke-basic-tiller/main.tf

```diff
@@ -330,6 +336,13 @@ resource "null_resource" "grant_and_configure_helm" {
 
     kubergrunt helm configure --helm-home ${pathexpand("~/.helm")} --tiller-namespace ${local.tiller_namespace} --resource-namespace ${local.resource_namespace} --rbac-user ${data.google_client_openid_userinfo.terraform_user.email} ${local.kubectl_auth_config}
     EOF
+
+    # Use environment variables for Kubernetes credentials to avoid leaking into the logs
+    environment = {
+      KUBECTL_SERVER_ENDPOINT = "${data.template_file.gke_host_endpoint.rendered}"
+      KUBECTL_CA_DATA         = "${base64encode(data.template_file.cluster_ca_certificate.rendered)}"
+      KUBECTL_TOKEN           = "${data.template_file.access_token.rendered}"
+    }
```

Comment:
Ah I must have forgotten this! Sorry about that! And it probably worked because we were configuring the kubeconfig, but definitely a ticking time bomb. Thanks for catching and fixing!

Reply:
@yorinasub17 took me a while to figure out what was going wrong ;)